### DISCUSSION TITLE
Challenges in calibrating TARSO-models: edges of the calibration parameter range and long response times

### DISCUSSION THREAD

**Discussion by strijkerb** [pastas/pastas#680] [Q&A] [answered]
Hi everyone,

I'm facing challenges calibrating some models, as the optimal parameter setting extends to the edges of the calibration parameter range. This is undesirable, as no local minimum has been found. Simultaneously, these models sufficiently explain the observations with R2 values above 0.7, but exhibit very large response times (t95=3000 days), which seem unrealistic.

Some context: I'm working with head levels in Dutch canal dikes, which can be very well explained by rainfall and evaporation on itself (water levels in the canal are almost constant). The heads are shallow and responsive, and exponential response functions seem to fit best. After evaluating various model structures, the TARSO-model gives the best results. I haven't used a noise model due to high-frequency measurements. Why does this model structure fit well? An explanation can be that shallow head levels result in different head responses near the surface and the hydraulic gradient in the dike, and therefore the head response, depends on the head itself.

As mentioned, the optimal model parameters are at the edge of the calibration range but still provide sufficient R2 values (>0.7). Why do models with that large response times align well with the observations? Is there another local minimum with different optimal parameter settings that the current calibration hasn't found? Any tips for these situations?

I've attempted adjusting initial parameter settings (A0, a0) and changing the solver, but it resulted in similar optimal values. The attached notebook includes measurements (head and meteo), exported models, and figures of various models. The notebook is divided into three parts:

0. Importing data
1. Some examples of TARSO-models to illustrate the problem
2. Background information where several model structures are investigated. This includes a table with r2-values for several model structures and the optimal model parameters of the calibrated TARSO-models.

I hope the notebook and my questions are clear. Thanks in advance.

Bart
[GitHub_calibration_TARSO.zip](https://github.com/pastas/pastas/files/14042861/GitHub_calibration_TARSO.zip)

**Comment by raoulcollenteur** [comment, marked as answer]
Hi @strijkerb,

Good to see Pastas used on this kind of data, and thanks for sharing the notebook. I guess the response is different above a certain head level, so that's why the TARSO model works better. I guess the parameters for the long response function are rather uncertain, but to figure that out you may want to consider using a noise model.. Otherwise long response times often means pastas is trying to fit some long-term trend in the model.

Hope this helps!

Cheers,
Raoul
